toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,67 @@
+"""Bures distance metric."""
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
+def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray ) -> float:
+    r"""
+    Compute the bures distance of two density matrices [WikFid]_.
+
+    Calculate the bures distance between the two density matrices:
+    `rho_1` and `rho_2`, defined by:
+    .. math::
+        \sqrt{2 (1 - F(\rho_1, \rho_2)},
+
+    where:`fidelity` denotes the fidelity between `rho_1` and `rho_2`
```

Comment:
where :math:\`F(\cdot)\` denotes the fidelity between :math:\`\rho_1\` and :math:\`\rho_2\`.